src/cats/labs/channel.cljc

```diff
@@ -78,64 +64,47 @@
 
     p/Functor
     (-fmap [_ f mv]
-      (let [c (a/chan 1 (map f))]
-        (a/pipe mv c)
-        c))
+      (a/pipe mv (a/chan 1 (map f))))
 
     p/Semigroup
     (-mappend [_ sv sv']
-      (chain-chans sv sv'))
+      (a/merge [sv sv']))
```

Comment:
Yes its a behavior change. I view channels more as streams or sets which are not necessarily sequential. If you were to `mappend` two infinite channels with `chain-chans`, you would never see values from the second channel, whereas `merge` can handle infinite channels, but doesn't give any sequential guarantees.